src/js/pyodide.ts

```diff
@@ -168,6 +169,11 @@ export async function loadPyodide(
      * Make loop.run_until_complete() function correctly using stack switching
      */
     enableRunUntilComplete?: boolean;
+    /**
+     * If false (default), throw an error if the version of Pyodide core does not
+     * match the version of the Pyodide js package.
```

Comment:
Could you update the default value and the comment correspondingly?

Reply:
I am not sure how to express it. Maybe could you suggest a change here?

Reply:
I think you can add `checkApiVersion: true` to default_config.

```js
  const default_config = {
    fullStdLib: false,
    jsglobals: globalThis,
    stdin: globalThis.prompt ? globalThis.prompt : undefined,
    lockFileURL: indexURL + "pyodide-lock.json",
    args: [],
    _node_mounts: [],
    env: {},
    packageCacheDir: indexURL,
    packages: [],
    enableRunUntilComplete: false,
    checkAPIVersion: true,
  };
```

Reply:
Oh I get it. I forgot that we have reversed the meaning of this options. My fault 🙏